File: app/src/main/java/com/forkmang/adapter/SpinnnerAdapter_Type_Value.java
package com.forkmang.adapter;

import android.annotation.SuppressLint;
import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.BaseAdapter;
import android.widget.TextView;

import com.forkmang.R;

public class SpinnnerAdapter_Type_Value extends BaseAdapter {
    Context context;
    String[] type_value;
    LayoutInflater inflter;

    public SpinnnerAdapter_Type_Value(Context applicationContext, String[] type_value) {
        this.context = applicationContext;
        this.type_value = type_value;
        inflter = (LayoutInflater.from(applicationContext));
    }

    @Override
    public int getCount() {
        return type_value.length;
    }

    @Override
    public Object getItem(int position) {
        return null;
    }

    @Override
    public long getItemId(int position) {
        return 0;
    }

    @Override
    public View getView(int position, View convertView, ViewGroup parent)
    {
        @SuppressLint({"ViewHolder", "InflateParams"})
        View view = inflter.inflate(R.layout.spinner_cell, null);
        TextView person = view.findViewById(R.id.txt_person);
        person.setText(type_value[position]);
        return view;
    }
}

File: app/src/main/java/com/forkmang/adapter/SpinnnerAdapter_Type_Value.kt
package com.forkmang.adapter

import android.annotation.SuppressLint
import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.BaseAdapter
import android.widget.TextView
import com.forkmang.R

class SpinnnerAdapter_Type_Value(var context: Context, var type_value: Array<String>) :
    BaseAdapter() {
    var inflter: LayoutInflater = LayoutInflater.from(context)

    override fun getCount(): Int {
        return type_value.size
    }

    override fun getItem(position: Int): Any {
        return type_value[position]
    }

    override fun getItemId(position: Int): Long {
        return 0
    }

    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        @SuppressLint("ViewHolder", "InflateParams") val view =
            inflter.inflate(R.layout.spinner_cell, null)
        val person = view.findViewById<TextView>(R.id.txt_person)
        person.text = type_value[position]
        return view
    }
}